### PR TITLE
Bumps PureScript to 0.11.7 (as a cabal dependency)

### PR DESCRIPTION
**Pull request by damncabbage**
~Just putting it up so we can muck around with it; not intended for merging (... yet?).~  
Done.

! @charleso @sphvn 
  
/jury approved @charleso

### CODE REVIEW

**Comment by damncabbage** [comment]
Finally got the 7.x builds done; onto the 8.x ones (mostly just lockfile bumping, AFAICT).

**Comment by damncabbage** [comment]
> All checks have passed
> 31 successful checks



requesting review please

File: loom-cli/test/cli/purs/library/loom.toml [hunk 1]
```diff
@@ -18,15 +18,33 @@
...
+    , ["purescript/purescript-tuples", "tags/v4.1.0", "9cace46d21e38243d97496a2cee9e46c0eb26f5a"]
+    , ["purescript/purescript-arrays", "tags/v4.2.2", "8d1aab654b4b15981ebd4a48066eb1797fbcd85b"]
+    , ["purescript/purescript-unfoldable", "tags/v3.1.0", "eb96bd95d54da9cb2b530ff94bcc54ab5eef309e"]
+    , ["purescript/purescript-nonempty", "tags/v4.1.1", "4978087f102f31154ca2afaaa25475f770a7d25e"]
+    , ["purescript/purescript-distributive", "tags/v3.0.0", "c887ffa3367ca38ca966ee3b4d44dab984c16bc5"]
+    , ["purescript/purescript-type-equality", "tags/v2.1.0", "51053827f5e8df45ac5e22d393cb04999ef433d0"]
+    , ["purescript/purescript-typelevel-prelude", "tags/v2.6.0", "6e12a8e4824bdd18e2ba33a4c082adbe03244228"]
+    , ["purescript/purescript-identity", "tags/v3.1.0", "a6c43400d870a6ac6b0d27592c08f38e37350ba5"]
+    , ["purescript/purescript-st", "tags/v3.0.0", "f94763242569154264f61d343739f73d3c5045c6"]
+    , ["purescript/purescript-eff", "tags/v3.1.0", "34682c32e0d4749bc8de9c0f92abe345fe7ca975"]
```
Comment (charleso) [inline loom-cli/test/cli/purs/library/loom.toml]: Just curious, why all these extra libraries just for the cli test? Are these the base standard for PS 0.11?

**Comment by charleso** [comment]
👍

File: loom-cli/test/cli/purs/library/loom.toml (hunk 1 above)
Comment (damncabbage) [reply loom-cli/test/cli/purs/library/loom.toml]: These kept coming up in build failures as being required by the test. It seems that the "core" libraries have a wider net of dependencies, AFAICT, yeah.

**Pull request merged**